burnman/minerals/other.py

```diff
@@ -12,10 +12,34 @@
 
 from .. import mineral_helpers as helpers
 from ..mineral import Mineral
+from ..solidsolution import SolidSolution
+from ..solutionmodel import *
 
 from .SLB_2011 import periclase, wuestite, mg_perovskite, fe_perovskite
 
-
+class make_endmember(SolidSolution):
```

Comment:
This not a typical naming for a class. Please use CamelCase and no underscore. The word "make" also sounds more like a function. Thinking about it, maybe you do want to create a function instead?
```
def make_endmember(mineral_list, stoichiometry_list, ESV):
  ss = SolidSolution(....)
  return ss
```